src/main/java/se/bjurr/gitchangelog/internal/git/GitRepo.java

```diff
@@ -47,10 +41,12 @@
   private Git git;
   private final Repository repository;
   private final RevWalk revWalk;
+  private final HashMap<String, GitRepo> submodules;
 
   public GitRepo() {
     this.repository = null;
     this.revWalk = null;
+    this.submodules = null;
```

Comment:
I would prefer an empty map here, not null

Reply:
Fixed.